frost-core/src/frost.rs

```diff
@@ -148,34 +148,67 @@ where
     }
 }
 
-/// Generates the lagrange coefficient for the i'th participant.
-#[cfg_attr(feature = "internals", visibility::make(pub))]
-fn derive_interpolating_value<C: Ciphersuite>(
-    signer_id: &Identifier<C>,
-    signing_package: &SigningPackage<C>,
+/// Generates a lagrange coefficient.
+///
+/// The Lagrange polynomial for a set of points (x_j, y_j) for 0 <= j <= k
+/// is ∑_{i=0}^k y_i.ℓ_i(x), where ℓ_i(x) is the Lagrange basis polynomial:
+///
+/// ℓ_i(x) = ∏_{0≤j≤k; j≠i} (x - x_j) / (x_i - x_j).
+///
+/// This computes ℓ_j(x) for the set of points `xs` and for the j corresponding
+/// to the given xj.
+///
+/// If `x` is None, it uses 0 for it (since Identifiers can't be 0)
+pub fn compute_lagrange_coefficient<C: Ciphersuite>(
+    xs: &BTreeSet<Identifier<C>>,
+    x: Option<Identifier<C>>,
```

Comment:
Before i start suggesting renames, what about `x_set` instead of `xs`, and `x_i` instead of `xi`?

Reply:
> Before i start suggesting renames, what about `x_set` instead of `xs`, and `x_i` instead of `xi`? (and then inside use `x_j`)

Reply:
That's indeed much better, changed in a3558ea